### DISCUSSION TITLE
I can't fork a repository

### DISCUSSION THREAD

**Discussion by michllean** [community/community#45979] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to fork this particular repository (https://github.com/LinkedInLearning/hands-on-javascript-2499547/fork) but I can't fork it. No error message, but the [Create fork] button is just disabled for me. My internet connection is secure. What may be the cause of this issue and what can I do to fix it?

**Comment by jge162** [comment]
When I click on your link I can fork.

**Reply by jge162** [reply]
Maybe they have a fork restriction.

**Reply by jge162** [reply]
Do you have access to LinkedIn learning? I have an account with them. Through school.

**Reply by michllean** [reply]
Yea I also have access through school but I still can't fork it for some reason.
It doesn't even show me an error message, so I'm not sure what's going on.

**Reply by jge162** [reply]
that is odd. IDK

**Comment by gdomingu** [comment]
I don't think this is a codespace question so I think the label should be removed, but make sure you "Select an owner" from the dropdown or the "Create fork" button will be disabled. 

<img width="522" alt="Screen Shot 2023-02-01 at 6 46 30 PM" src="https://user-images.githubusercontent.com/4443635/216218669-d59b7693-f49c-491a-a016-97587225d3c2.png">

**Reply by cristianc** [reply]
Where does this screen come from?

Waiting for a reply,

Thanks.

**Comment by ArMaxik** [comment]
Do you have ad blocking extentions? Disable them and try again
